students/ltrisolini/Lesson09/mailroom_OO.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+import sys
+from collections import defaultdict
+
+class Donor:
+
+    def __init__(self, name, donations=None):
+        self.name = name
+        if donations is None:
+            self.donations = []
+        else:
+            self.donations = donations
+
+
+    def add_donation(self, amount):
+        try:
+            self.donations.append(int(amount))
+        except ValueError:
+            print("Please enter a number!")
+
+
+    def number_donations(self):
+        return len(self.donations)
+
+
+    def total_donation(self):
```

Comment:
method in a class should have 1 line spacing between them (module functions have 2 lines space)